Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM mjstealey/hs_docker_base:1.6.5
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
+RUN pip uninstall -y django-celery
+RUN pip install -U kombu==3.0.33
+RUN pip install -U celery==3.1.20

```

Comment:
@mjstealey @selimnairb I referenced @selimnairb's rehesys implementation branch to replace django-celery with the latest celery version which supports django out of the box. @mjstealey you can change the hs_base image accordingly, then we don't need these three lines.